package.nls.json

```diff
@@ -105,6 +105,7 @@
     "azureFunctions.viewCommitInGitHub": "View Commit in GitHub",
     "azureFunctions.viewDeploymentLogs": "View Deployment Logs",
     "azureFunctions.viewProperties": "View Properties",
+    "azureFunctions.eventGrid.sendMockRequest": "Save and send request...",
```

Comment:
Wondering if we should call this `Save and execute`? I know technically it's sending the request to the local host, but the entry point is call `Execute Function Now...` so it may be somewhat confusing verbiage.

Reply:
Done.